tmt/cli.py

```diff
@@ -530,6 +530,41 @@ def finito(
         click_context.obj.run.go()
 
 
+def _apply_linters(lintable: Union[
+            tmt.lint.Lintable[tmt.base.Test], 
+            tmt.lint.Lintable[tmt.base.Plan], 
+            tmt.lint.Lintable[tmt.base.Story], 
+            tmt.lint.Lintable[tmt.base.LintableCollection]], 
+            linters: list[tmt.lint.Linter],
+            failed_only: bool,
+            enforce_checks: list[str],
+            outcomes: list[tmt.lint.LinterOutcome],) -> tuple[bool, Union[list[tmt.lint.LinterRuling], None]]:
+        """ 
+        Local function applying linters on a lintable instance and filter the
+        resulting rulings based on users desired ones.
+        """
```

Comment:
Docstring should have a short, single line at its beginning, e.g. "Apply linters on a lintable and filter out disallowed outcomes"?